routes/service.php

```diff
@@ -130,4 +130,26 @@
         'as' => 'admin.deliveries.store',
         'uses' => 'Admin\DeliveriesController@store',
     ]);
+
+    // Market Management
+    Route::get('markets', [
+        'as' => 'admin.markets.index',
+        'uses' => 'Admin\MarketsController@index',
+    ]);
+    Route::get('markets/create', [
+        'as' => 'admin.markets.create',
+        'uses' => 'Admin\MarketsController@create',
+    ]);
+    Route::post('markets', [
+        'as' => 'admin.markets.store',
+        'uses' => 'Admin\MarketsController@store',
+    ]);
+    Route::get('markets/{id}/edit', [
```

Comment:
would we want to constrain the id param? https://laravel.com/docs/6.x/routing#parameters-regular-expression-constraints

Reply:
oh! interesting!
have made a card as we might want to do that all over the place.